src/components/components/ebay-notice-base/index.marko

```diff
@@ -48,6 +50,11 @@ $ var prefixClass = input.prefixClass;
             </else-if>
         </>
     </if>
+    <if(input.cta)>
+        <p class="page-notice__cta">
+            $!{input.cta}
+        </p>
```

Comment:
Oh we should use attribute tags instead of this. This is unsafe to do. 

You will need to add the definition in `marko-tag.json` to get it to work. 

By doing an attribute tag, we could actually just do:

`<@cta href="link">Body text</@cta>`

Which is pretty nice. 

We can do:
```
<if(input.cta)>
        <p class="page-notice__cta">
          <a ...input.cta><${input.cta}/></a>
       </p>
```

We could also override the `<a>` if needed, but I think it should be fine for now.

Reply:
Yea, I was wondering about this. I realized unescaped HTML may not be the best option. Ok, let me work on changing this.

Reply:
Ok, I made the switch. I wasn't sure about the changes in `marko-tag.json` since I don't know what those options mean; was just a copy/paste/modification.